api/worker/proto/worker_service.proto

```diff
@@ -40,9 +40,18 @@ message StopRequest {
     bool force = 2;
 }
 
+message StreamRequest {
+    // The Job to be stopped.
+    Job job = 1;
+    // the nb of sequence of the starting point of the stream
+    int32 startPoint = 2;
+}
+
 message Log {
+    // Nb of sequence of the current entry in the stream
+    int32 nbSequence = 1;
```

Comment:
I don't beleve this was documented in the design document, why not just start the stream from the beginning?

Reply:
Hi, I think that I have mentioned it in the streaming design section. The connection backoff is for long-live streaming